test/widgets/content_test.dart

```diff
@@ -1158,4 +1158,19 @@ void main() {
       check(linkText.textAlign).equals(TextAlign.center);
     });
   });
+
+  group('ListNodeWidget', () {
+    testWidgets('ordered list with custom start', (tester) async {
+      await prepareContent(tester, plainContent('''
+        <ol start="3">
+          <li>third</li>
+          <li>fourth</li>
+        </ol>'''));
```

Comment:
nit: Use single-quouted strings
```suggestion
      await prepareContent(tester, plainContent('<ol start="3">\n<li>third</li>\n<li>fourth</li>\n</ol>'));
```